src/components/navigation/LinkUnstyled.js

```diff
@@ -0,0 +1,31 @@
+import classnames from 'classnames';
+
+import { downcastRef } from '../../util/typing';
+
+import LinkBase from './LinkBase';
+
+/**
+ * Render a link (`<a>`) with common attributes but no styling
+ *
+ * @typedef {import('../../types').PresentationalProps} CommonProps
+ * @typedef {import('preact').JSX.HTMLAttributes<HTMLAnchorElement>} HTMLAnchorAttributes
+ *
+ * @param {CommonProps & HTMLAnchorAttributes} props
+ */
+export default function LinkUnstyled({
```

Comment:
Basically a wrapper around `LinkBase` but disallowing `className` (so, shaped like the presentational component it is).